### PR TITLE
Feed Verification on dbot for Responsive

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1739
Desc: Add check to verify feed on dbot using responsive.

### CODE REVIEW

File: cypress/e2e/wip/dbot/sanity-tests/FeedVerification.cy.js [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
 describe('QATEST-99340: Verify feed is loading on charts tab', () => {
   const tradersHub = new TradersHub()
-
+  const size = ['small', 'desktop']
   beforeEach(() => {
     if (Cypress.config().baseUrl === Cypress.env('prodURL')) {
       cy.c_login({ user: 'dBot', rateLimitCheck: true })
     } else {
-      cy.c_visitResponsive('/')
       cy.c_createRealAccount()
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/dbot/sanity-tests/FeedVerification.cy.js:11]: Bertrand needs to make a minor change to c_createRealAccount. So, you'd have to wait until this is completed unfortunately.

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/dbot/sanity-tests/FeedVerification.cy.js (hunk 1 above)
Comment (markodonnell-deriv) [reply cypress/e2e/wip/dbot/sanity-tests/FeedVerification.cy.js:11]: Disregard this comment now, as discussed :)

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**